### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: cdap-ui/README.rst [hunk 1]
```diff
@@ -44,40 +41,31 @@ Unix/Linux Flavors
...
 ===========================================
-
-1. Upon installing the UI Pack, go to Cask Market
+1. After installing the CDAP UI Pack and restarting CDAP, from within the CDAP UI go to the Cask Market
 2. From the **Solutions** category, follow the steps for the **Data Preparation** solution
-3. Go to Data Preparation by clicking on the CDAP menu and then choosing **Data Preparation**
-4. If newer version of the Data Preparation libraries have been installed, the UI will show an **Update** button.
-5. Click the **Update** button to enable the newer version of Data Preparation.
+3. Go to *Data Preparation* by clicking on the CDAP menu and then choosing *Data Preparation*
+4. If a newer version of the *Data Preparation* libraries has been released, the UI will show an **Update** button
```
Comment (bdmogal) [inline cdap-ui/README.rst]: its not released, its installed - the user installed the artifact from the Cask Market.

**Review by bdmogal — COMMENTED** [review]
One comment, then LGTM

File: cdap-ui/README.rst (hunk 1 above)
Comment (awholegunch) [reply cdap-ui/README.rst]: I'll have to take your word for it, as step 2 could cover that, and I don't have a way to test that...

**Comment by bdmogal** [comment]
LGTM

**Pull request merged**